src/topics/__tests__/topicsSelectors-test.js

```diff
@@ -112,21 +106,16 @@ describe('getTopicsForStream', () => {
         ],
       },
       mute: [['stream 1', 'topic 1'], ['stream 1', 'topic 3'], ['stream 2', 'topic 2']],
-      unread: {
-        ...eg.baseReduxState.unread,
-        streams: [
-          {
-            stream_id: 1,
-            topic: 'topic 2',
-            unread_message_ids: [1, 5, 6],
-          },
-          {
-            stream_id: 1,
-            topic: 'topic 4',
-            unread_message_ids: [7, 8],
-          },
-        ],
-      },
+      unread: [
+        eg.streamMessage({ stream_id: 1, subject: 'topic 2', id: 1 }),
+        eg.streamMessage({ stream_id: 1, subject: 'topic 2', id: 5 }),
+        eg.streamMessage({ stream_id: 1, subject: 'topic 2', id: 6 }),
+        eg.streamMessage({ stream_id: 1, subject: 'topic 4', id: 7 }),
+        eg.streamMessage({ stream_id: 1, subject: 'topic 4', id: 8 }),
+      ].reduce(
+        (st, message) => unreadReducer(st, mkMessageAction(message)),
+        eg.baseReduxState.unread,
```

Comment:
Would the `initialState` export from unread-testlib.js do just as well as `eg.baseReduxState.unread`, or should we stick with the latter?

Reply:
Ah, yes. I think it was as I was fixing up this test file (after writing most of the other changes) that I realized that `eg.baseReduxState.unread` should be exactly equivalent to that `initialState` export -- after all, the point of the way we construct that `initialState`, with `{ type: eg.randString() }`, is that it exercises exactly the same path through the reducer that Redux uses to initialize the state. (And `eg.baseReduxState` comes via Redux's `createStore`.)

I'm not entirely sure what the most helpful style is for referring to that state.
* Here, `eg.baseReduxState.unread` feels natural because it fits in with the rest of this global state we're assembling.
* In the unread-model tests (including `unreadSelectors-test.js`), `initialState` or `initialUnreadState` feels natural because it makes it clear that it's local -- these are the tests of this very code, so they shouldn't need to appeal to some central library to get an instance of this state.

I'm inclined to leave the style different between these places at least for now, and perhaps in the future we'll settle on one of them.

Reply:
> I'm inclined to leave the style different between these places at least for now, and perhaps in the future we'll settle on one of them.

Sounds good; I don't have strong feelings either way.